steps/github-step-repositories-describe/spec.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"type": "object",
+	"properties": {
+		"token": {
+			"type": "string",
+			"description": "A secret containing a token with the appropriate permissions."
+      "writeOnly": true
+		},
+		"org": {
+			"type": "string",
+			"description": "The GitHub org to query."
+		},
+    "scope": {
+      "type": "string",
+      "description": "The repository scope or type to query for. (all, public, private, forks, sources, member)",
```

Comment:
JSON schema supports an `enum` field that you could potentially use here, although I might caution against it because it will prevent "forward-compatible" updates to the GitHub API that would allow users to query for new scopes without changing the step definition. It's sometimes better to link to docs that have a list of valid values instead, or make sure you include some language that makes it clear the list of options isn't exhaustive.

Reply:
with this and the next comment, I'm not completely following. Should I switch this, the example, and the step code to all use a connection, all use a secret/token, or some other combination?